index.js

```diff
@@ -1,1144 +0,0 @@
-  } else if (inputArr.includes("i")) {
-    console.log("\nYou are currently in possesion of: " + player.inventory);
-    playKitchen();
-  } else if (inputArr.includes("move") && inputArr.includes("body")) {
-    deadBody.action();
-    console.log(body.extraDesc);
-    playKitchen();
-  } else if (inputArr.includes("speak") && inputArr.includes("green")) {
-    console.log(green.dialogueOne);
-
-    if (!player.hasSpokenTo.includes("green")) {
-      player.hasSpokenTo.push("green");
-    }
-    playKitchen();
-  } else if (inputArr.includes("take") && inputArr.includes("pan")) {
-    takeItem(kitchen,"Frying Pan");
-    playKitchen();
-  } else if (inputArr.includes("take") && inputArr.includes("notebook")) {
-    if (player.hasMoved === true) {
-      takeItem(kitchen, "Notebook");
-    } else {
-      console.log("\nThere is no notebook to be seen.");
-    }
-    playKitchen();
-  } else if (inputArr.includes("examine") && inputArr.includes("notebook")) {
-    if (player.inventory.includes("Notebook")) {
-      console.log(notebook.desc);
-      playKitchen();
-    } else {
-      console.log("\nYou are not in possesion of a notebook.");
-      playKitchen();
-    }
-  } else if (inputArr.includes("take") && inputArr.includes("key")) {
-    takeItem(kitchen, "Key");
-    playKitchen();
-  } else if (inputArr.includes("use") && inputArr.includes("stove")) {
-    stove.action();
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("pan")) {
-    dropItem(kitchen, "Frying Pan");
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("notebook")) {
-    dropItem(kitchen, "Notebook");
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("key")) {
-    dropItem(kitchen, "Key");
-    playKitchen();
-  } else if (inputArr.includes("go") && inputArr.includes("dining")) {
-    player.location = "The Dining Room";
-    console.log(diningMessage);
-    playDining();
-  } else if (inputArr.includes("go") && inputArr.includes("conservatory")) {
-    player.location = "The Conservatory";
-    console.log(conservatoryMessage);
-    playConservatory();
-  } else if (inputArr.includes("go") && inputArr.includes("library")) {
-    player.location = "The Library";
-    console.log(libraryMessage);
-    playLibrary();
-  } else if (inputArr.includes("go") && inputArr.includes("study")) {
-    player.location = "The Study";
-    console.log(studyMessage);
-    playStudy();
-  } else if (inputArr.includes("go") && inputArr.includes("lounge")) {
-    player.location = "The Lounge";
-    console.log(loungeMessage);
-    playLounge();
-  } else if (inputArr.includes("go") && inputArr.includes("billiard")) {
-    player.location = "The Billiard Room";
-    console.log(billiardMessage);
-    playBilliardRoom();
-  } else if (inputArr.includes("go") && inputArr.includes("ballroom")) {
-    player.location = "The Ballroom";
-    console.log(ballroomMessage);
-    playBallroom();
```

Comment:
Including all of these options on each play*Room* function means that the player can freely move through the house. Either removing the options of rooms not attached to the kitchen, or by creating a state machine and defining which rooms can move to other rooms would make this functionality better

Reply:
We would rather have the character be able to move freely as they follow the narrative to cut down on unnecessary move commands and focus more on being able to inspect the rooms/items/characters. Re-reading the project's stories it doesn't seem like a state machine was a necessary story so we'd rather skip that if it's all the same.

We did, however, combine all the player logic into a single play function which saved about 400 lines of code!